public/ipc.js

```diff
@@ -157,17 +155,39 @@ ipc.on('get job', async (event, data) => {
 })
 
 ipc.on('abort job', (event, data) => {
-  Object.values(queues[data.job.queue.name].childPool.retained).forEach((child) => {
-    child.send({__abortJobId__: data.job.id})
-  })
+  _dialog.showMessageBox(
+    {
+      type: 'question',
+      buttons: ['Cancel', 'Abort'],
+      message: `Are you sure you want to abort the selected ${data['job']['name']} job?`
```

Comment:
Could you post a screenshot of this dialog please?

Reply:
<img width="1503" alt="Screen Shot 2020-01-27 at 3 26 29 PM" src="https://user-images.githubusercontent.com/10591665/73211027-6d2ee880-4119-11ea-97ff-d099f1336f2c.png">